modules/javafx.graphics/src/test/java/test/com/sun/javafx/pgstub/StubTextLayout.java

```diff
@@ -33,6 +33,12 @@
 import javafx.scene.shape.PathElement;
 import javafx.scene.text.Font;
 
+/**
+ * Stub implementation of the {@link TextLayout} for testing purposes.
+ * <br>
+ * Can calculate the bounds of text by simply using the size of the font.
+ * If the text is bold, the font will be 1 pixel bigger.
```

Comment:
maybe change to "somewhat wider" instead, since the height is the same?

Reply:
sure!
Fun fact: I also was not happy with the javadoc but couldn't come up with a better wording. Fixed now. :)